src/emulator/CMakeLists.txt

```diff
@@ -49,13 +49,16 @@ endmacro(configure_boost)
 
 configure_boost()
 
+# add_definitions(-DUSE_GDBSTUB)
+
 add_subdirectory(audio)
 add_subdirectory(cpu)
 add_subdirectory(crypto)
 add_subdirectory(ctrl)
 add_subdirectory(dialog)
 add_subdirectory(disasm)
 add_subdirectory(glutil)
+add_subdirectory(gdbstub)
```

Comment:
Put this before ```glutil```. Everything is in order and it should be, PetMac wants this to prevent conflicts in the future.

a b c **d** e f g h i j k **l** m n o p q r s t u v w x y z